### ISSUE TITLE
Integrate gcloud setup workflows for ViUR projects

### ISSUE THREAD

**Issue by phorward** [viur-framework/viur-cli#4]
This issue is a collection currently resolved in several scripts, which should be merged into viur-cli under a uniform interface / command, e.h. `viur-cli maintain`.

- [x] `viur-cli setup all` should perform the operations from [`viur-gcloud-setup.sh`](https://github.com/viur-framework/viur-base/blob/main/viur-gcloud-setup.sh)
- [x] #70
- [x] `viur-cli setup index` for index.yaml sorting and deployment (see #2 as well, maybe can be integrated with this)
- [x] `viur-cli setup queue` for queue.yaml deployment
- [x] `viur-cli setup cron` for cron.yaml deployment
- [x] `viur-cli setup cicd` for setting-up and update CI/CD workflows

**Comment by Grashalmbeisser** [comment]
The operations from  #https://github.com/viur-framework/viur-base/blob/main/viur-gcloud-setup.sh is handled in the new cloud.py. 
The Base for CiCd is Pr'd in #https://github.com/viur-framework/viur-base/pull/125 and wont be fixed in the CLI for now

The issue should be closed after #98 https://github.com/viur-framework/viur-cli/pull/98
